std/meta.d

```diff
@@ -301,29 +301,19 @@ template staticIndexOf(alias T, TList...)
 
 // [internal]
 private template genericIndexOf(args...)
-if (args.length >= 1)
+    if (args.length >= 1)
 {
-    alias e     = OldAlias!(args[0]);
-    alias tuple = args[1 .. $];
-
-    static if (tuple.length)
+    static foreach (idx, argi; args[1 .. $])
```

Comment:
Nit: Why not "arg" instead of "args" (it's a typical convention to denote the array with a "s" postfix.

Reply:
Done.